megamek/src/megamek/common/Mech.java

```diff
@@ -350,6 +362,18 @@ public Mech(int inGyroType, int inCockpitType) {
         case COCKPIT_QUADVEE:
             setCrew(new Crew(CrewType.QUADVEE));
             break;
+        case COCKPIT_SUPERHEAVY_COMMAND_CONSOLE:
```

Comment:
Don't repeat the same code for multiple cases. Add the additional cases to the one that's already there:
>>>
        case COCKPIT_SUPERHEAVY_COMMAND_CONSOLE:
        case COCKPIT_COMMAND_CONSOLE:
        case COCKPIT_SUPERHEAVY_COMMAND_CONSOLE:
        case COCKPIT_SMALL_COMMAND_CONSOLE:
            setCrew(new Crew(CrewType.COMMAND_CONSOLE));
            break;